net/acme/files/run.sh

```diff
@@ -54,6 +54,11 @@ pre_checks()
     log "Running pre checks for $main_domain."
 
     listeners="$(get_listeners)"
+	
+	if [ $(grep $listeners "\.") ]; then
+		listeners=$(echo $listeners | sed "s/\..*//")
+	fi
```

Comment:
What's this supposed to do?

Reply:
i notice nginx proess is displayed with the config name (nginx.config) this "fix this to have a better name than nginx.config"